packages/nextjs-contentful-renderer/src/configNavbar/configNavbar.ts

```diff
@@ -0,0 +1,40 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import { Block, Model, Page } from 'src/utils/contentful/getPageById/types';
+import { getDefaultNavbar } from '../utils/contentful/getDefaultNavbar';
+
+interface QueryConfig {
+  apolloClient: ApolloClient<NormalizedCacheObject>;
+  domain: string | undefined;
+  preview: boolean;
+}
+
+export const configNavbar = async (
+  page: Partial<Page>,
+  { apolloClient, domain, preview }: QueryConfig
+): Promise<Block[]> => {
+  const pageNavbar = page?.navbar;
+  const content = page?.content;
+
+  if (pageNavbar) {
+    return concatNavbar(content, pageNavbar);
+  }
+
+  const defaultUniversalNavbar = await getDefaultNavbar(apolloClient, domain, preview);
+
+  if (!defaultUniversalNavbar) {
+    return content;
+  }
```

Comment:
this is unnecessary, as we discussed in slack a page might not have a navbar

Reply:
On the 18 i am validating if the page. has its own `navbar`, and if not, then I get the default navbar by `domain`, but if there is no a default navbar or that response is null or empty, I return the original content without any change, that is, without any navbar associated to it.

Reply:
And this is wrong. As we discussed in slack If a page doesn’t have a nav bar then no nav bar should be rendered

Reply:
@tkanzakic I removed that feature, and just return a content with a navbar as long as it exists in the page